### DISCUSSION TITLE
Expand multiple Columns

### DISCUSSION THREAD

**Discussion by bastigerner** [Megabit/Blazorise#1876] [Q&A]
Hey,

didn't find anything on that yet, but is it possible to expand multiple columns inside a DataGrid?

Best Regards,
Sebastian

**Comment by stsrki** [comment]
Did you mean to expand the **detail row** maybe?

It is controlled by with `DetailRowTrigger`, eg. 

`DetailRowTrigger="@((item)=>item.Salaries?.Count > 0 && item.Id == selectedEmployee?.Id)"`.

As long as it returns true it, the row that triggered it will be expanded. If you want for all to expand just do

 `DetailRowTrigger="@((item)=>true)"`

**Reply by bastigerner** [reply]
Yes i meant that, thanks for the fast reply.
Pretty simple thought so but could test it atm. 😄
